<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nguyễn Võ Như Ngọc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>using string processing functions</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Input data:</string>
    </property>
    <widget class="QLineEdit" name="lineEditInputdata">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Output data:</string>
    </property>
    <widget class="QLineEdit" name="lineEditOutputdata">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonInputdata">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUppercase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Uppercase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLowercase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lowercase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCountlowercasecharacters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Count lowercase characters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCountuppercaseletters">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Count uppercase letters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrintnumberofwordsperline">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Print number of words per line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWordcount">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Word count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrintvowelsconsonants">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Print vowels, consonants</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
